rasa/test.py

```diff
@@ -109,7 +109,12 @@ def test_core(
     )
 
 
-def test_nlu(model: Optional[Text], nlu_data: Optional[Text], kwargs: Optional[Dict]):
+def test_nlu(
+    model: Optional[Text],
+    nlu_data: Optional[Text],
+    kwargs: Optional[Dict],
+    output: Text = DEFAULT_RESULTS_PATH,
```

Comment:
Minor: I would like to keep `kwargs` as last argument. Can you change the order?